build/xtask/src/dist.rs

```diff
@@ -2082,7 +2248,7 @@ pub fn make_kconfig(
     for (i, (name, task)) in toml.tasks.iter().enumerate() {
         let stacksize = task.stacksize.or(toml.stacksize).unwrap();
 
-        let flash = &task_allocations[name]["flash"];
+        let flash = &task_allocations[name]["flash"][0];
```

Comment:
I kinda feel like every case where we're explicitly accessing the first element of the list should get a comment, because each time I stop and go "uh... wait... is this right or making an assumption"

Reply:
Yup, fixed with `ContiguousRanges` in 3d128cbee0fd75fab17799a754281d836d4a7321